CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### 🚀 New components 🚀
+
+- (Splunk) Add RabbitMQ receiver ([#4980](https://github.com/signalfx/splunk-otel-collector/pull/4980))
```

Comment:
- Are there any limitations?
- How does this compare to the `collectd/rabbitmq` smartagent receiver?

Reply:
This is the native RabbitMQ. I believe the docs team will document this. I am not aware of the collectd/rabbitq to be honest. But the goal is oneday to remove from it.

Reply:
Also like other native receivers that we have brought in ( for e.g. Mongodb, MySQL), usually docs team reflect the offical contrib documentation of the respective receiver.
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/rabbitmqreceiver